site.config.ts

```diff
@@ -21,9 +21,9 @@ export default siteConfig({
   //twitter: '#',
   github: 'conrad1451',
   linkedin: 'conrad-hansen-quartey',
-  // mastodon: '#', // optional mastodon profile URL, provides link verification
-  // newsletter: '#', // optional newsletter URL
-  // youtube: '#', // optional youtube channel name or `channel/UCGbXXXXXXXXXXXXXXXXXXXXXX`
+  mastodon: '#', // optional mastodon profile URL, provides link verification
```

Comment:
Uncommenting the mastodon line, newsletter, and youtube caused the deployment to fail. So I will not merge this branch with main. Good this branch was tested via deployment.